### DISCUSSION TITLE
Can't find ImageMagick-6.6.9-Q16

### DISCUSSION THREAD

**Discussion by AngelosMako** [ImageMagick/ImageMagick#7782] [Help] [answered]
### ImageMagick version

ImageMagick-6.6.9-Q16

### Operating system, version and so on

Windows

### Description

Hello, I am working on remaking an old game of my, and my art pipeline has ImageMagick-6.6.9-Q16, but I can't find it in the releases. 
Tried running winget, 
winget install --id ImageMagick.ImageMagick.Q16 --version 6.6.9
winget install --id ImageMagick.ImageMagick --version 6.6.9
winget install --id ImageMagick.ImageMagick.Q16 --version 6.6.9.10
winget install --id ImageMagick --version 6.6.9

Everytime, I got "No version found matching 6.6.9", 
Any help is appreciated!

**Comment by AngelosMako** [comment]
From looking into it more, legacy cannot be installed from winget. 
Could it be that I have to build from source to get the exact version I want?

**Comment by snibgo** [comment]
I know nothing about winget.

I suggest upgrading to v7.

IM v6.6.9 is old, really old. Why do you want to install such old software?

For Windows, the current v6 binary (v6.9.13-19) is available at https://legacy.imagemagick.org/script/download.php#windows . Sources for some older versions are available at https://imagemagick.org/archive/ . But not as old as v6.6.9.

**Reply by AngelosMako** [reply]
That is what the pipeline was built around with!
It was made back in 2011.

**Comment by dlemstra** [comment, marked as answer]
We will not be going to publish a winget package for ImageMagick 6. You should switch to ImageMagick 7 instead.
